src/main/java/com/slimebot/util/Util.java

```diff
@@ -42,4 +47,51 @@ public static boolean isValidURL(String url) {
 			return false;
 		}
 	}
+
+	public static Instant timestamp() {
+		return new Date().toInstant().atZone(ZoneId.systemDefault()).toInstant();
+	}
+
+	public static String padRight(String s, int n) {
+		return padRight(s, ' ', n);
+	}
+
+	public static String padLeft(String s, int n) {
+		return padLeft(s, ' ', n);
+	}
+
+	public static String padRight(String s, char padChar, int n) {
```

Comment:
Make padChar String

Reply:
why? it must be a char that can be perfectly adapted to the specified length